sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/CassandraResourcesOperations.cs

```diff
@@ -2790,13 +2790,14 @@ public async Task<AzureOperationResponse<ThroughputSettingsGetResults>> BeginMig
             string _responseContent = null;
             if ((int)_statusCode != 200 && (int)_statusCode != 202)
             {
-                var ex = new ErrorResponseUpdatedFormatException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
+                var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
```

Comment:
This will introduce a breaking change.  Please have the breaking change team add the Approved-BreakingChange label to the spec PR https://github.com/Azure/azure-rest-api-specs/pull/13244

Reply:
Oh no, not again. This is a place where our preview and release Swagger versions are out of sync. I guess I'll go force it back to being an ErrorResponseUpdatedFormat and regenerate the C#.

Reply:
I should have pointed you to https://github.com/Azure/azure-rest-api-specs/pull/12861, which was the previous PR (a bug slipped through and had to get fixed). That PR has the Approved-BreakingChange label on it.